system_tests/espresso_e2e_test.go

```diff
@@ -362,37 +464,43 @@ func TestEspressoE2E(t *testing.T) {
 		return condition
 	})
 	Require(t, err)
-	err = waitForWith(
-		t,
-		ctx,
-		time.Minute*10,
-		time.Second*10,
-		func() bool {
-			log.Info("good staker acts", "step", i)
-			txA, err := goodStaker.Act(ctx)
-			Require(t, err)
-			if txA != nil {
-				_, err = builder.L1.EnsureTxSucceeded(txA)
-				Require(t, err)
-			}
-
-			log.Info("bad staker acts", "step", i)
-			txB, err := badStaker.Act(ctx)
-			if txB != nil {
-				_, err = builder.L1.EnsureTxSucceeded(txB)
-				Require(t, err)
-			}
-			if err != nil {
-				ok := strings.Contains(err.Error(), "ERROR_HOTSHOT_COMMITMENT")
-				if ok {
-					return true
-				} else {
-					t.Fatal("unexpected err")
-				}
-			}
-			i += 1
-			return false
 
-		})
-	Require(t, err)
+	// The following tests are very time-consuming and, given that the related code
+	// does not change often, it's not necessary to run them every time.
+	// Note: If you are modifying the smart contracts, staker-related code or doing overhaul,
+	// please manually uncomment the following and ensure the tests pass locally.
```

Comment:
I think it would be better if we execute them conditionally (for example if a certain env var is set) so we can at least run them on the CI, and one doesn't need to edit code to run them locally.

Reply:
good idea, I can condition this on an env var